app/assets/stylesheets/stores/_store_new_product.scss

```diff
@@ -236,3 +239,12 @@ $title-font: #353434;
 .link-button-text {
   padding-top: 7px;
 }
+
+.alert {
+  margin: 0 auto;
+  background-color: green
+  &__message {
```

Comment:
Expected indentation of 4 spaces           indentation<br>Expected a trailing semicolon              declaration-block-trailing-semicolon<br>Unexpected whitespace at end of line       no-eol-whitespace